roles/node/templates/mainnet/systemd.service.j2

```diff
@@ -31,7 +31,7 @@ ExecStart={{ node_bin_dir }}/{{ service_identifier }} \
 {% if solana.validator.limit_ledger_size | default('', true) | trim != '' %}
 --limit-ledger-size {{ solana.validator.limit_ledger_size }} \
 {% endif %}
-{% if solana.validator.no_snapshot_fetch | default(false) %}
+{% if not snapshot_fetch | default(false) %}
```

Comment:
can we please rename the variable to fetch_snapshot, sounds better for a yes no questions